### ISSUE TITLE
Default Outline Form Field State Color Contrast

### ISSUE THREAD

**Issue by ianbever** [angular/material.angular.io#728]
Regarding the accessibility built into Angular Material's components, I see that the default color for the outline form field is a 1px rgba(0,0,0,0.12) border that doesn't meet a 3:1 contrast ratio with a white surface to meet WCAG 2.1 Success Criterion 1.4.11: Non-text Contrast.

https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

Is it possible to darken this to black at rgba(0,0,0,0.42), which would meet the minimum WCAG color contrast for non-text elements? From there, there could still be adequate contrast between rgba(0,0,0,0.42) and whatever color is selected for the form field outline's focus state.

**Comment by Splaktar** [comment]
Please submit Angular Material and CDK questions [here](https://groups.google.com/forum/#!forum/angular-material2) and issues [here](https://github.com/angular/components/issues). This repo is for docs infrastructure only.

I'm moving this to the proper repo.
